paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -258,12 +260,24 @@ public class PaparazziPlugin @Inject constructor(
         test.doFirst {
           // Note: these are lazy properties that are not resolvable in the Gradle configuration phase.
           // They need special handling, so they're added as inputs.property above, and systemProperty here.
+          val isVerificationTask = isVerifyRun.get()
+
           test.systemProperties["paparazzi.layoutlib.runtime.root"] =
             layoutlibNativeRuntimeFileCollection.singleFile.absolutePath
           test.systemProperties["paparazzi.layoutlib.resources.root"] =
             layoutlibResourcesFileCollection.singleFile.absolutePath
           test.systemProperties["paparazzi.test.record"] = isRecordRun.get()
-          test.systemProperties["paparazzi.test.verify"] = isVerifyRun.get()
+          test.systemProperties["paparazzi.test.verify"] = isVerificationTask
+
+          if (isVerificationTask && snapshotOutputDir.asFile.exists().not()) {
+            test.logger.log(
+              WARN,
+              """
+                Snapshot directory not found: ${snapshotOutputDir.asFile.path}
+                Please run the `recordPaparazzi$variantSlug` task to generate snapshots.
+              """.trimIndent()
+            )
```

Comment:
cc @jrodbx I added a warning log for cases where the verify task runs without a snapshot directory created.